apple/internal/apple_framework_import.bzl

```diff
@@ -356,10 +374,21 @@ def _apple_static_framework_import_impl(ctx):
 
     return providers
 
+_common_attrs = {
+    "xcframework_platform_paths": attr.string_dict(
```

Comment:
This can be a bit simpler if we use platform IDs instead. For example:

```starlark
apple_dynamic_framework_import(
    name = "AmazonChimeSDK",
    framework_imports = glob(["AmazonChimeSDK.xcframework/**"]),
    visibility = ["//visibility:public"],
    xcframework_platform_ids = {
        "iphoneos": "ios-arm64_armv7",
        "iphonesimulator": "ios-i386_x86_64-simulator",
    },
)
```

Reply:
🤔  yeah not a bad idea

Reply:
Hello Zach, how is it going with this change?